automatic/clementine/tools/chocolateyInstall.ps1

```diff
@@ -1,30 +1,27 @@
-$ErrorActionPreference = 'Stop'
+﻿$ErrorActionPreference = 'Stop';
```

Comment:
it was just automatically added, I'll remove it. I'll do the same with the other one.